src/hotspot/cpu/s390/stubGenerator_s390.cpp

```diff
@@ -1713,6 +1749,10 @@ class StubGenerator: public StubCodeGenerator {
     const Register srclen  = Z_ARG2; // Must be odd reg and pair with src. Overwrites destination address.
     const Register dst     = Z_ARG3; // Must be even reg (KM requirement). Overwrites expanded key address.
 
+    if (! VM_Version::has_Crypto_AES()) {
+      __ should_not_reach_here();
+    }
+
```

Comment:
Please assert that VM_Version::has_Crypto_AES() instead. Otherwise, the stub shouldn't get generated at all (handled by vm_version_s390.cpp).